packages/webapp/src/credits/constants.js

```diff
@@ -1,6 +1,8 @@
 const routes = {
   ESTIMATOR_CREDITS_COST: '/credits-estimation/credits-cost',
-  ESTIMATOR_CREDITS_TIER: '/credits-estimation/credits-tier'
+  ESTIMATOR_CREDITS_TIER: '/credits-estimation/credits-tier',
+  ESTIMATOR_CREDITS_TERM_AND_CONDITIONS: '/credits-estimation/credits-terms-and-conditions',
```

Comment:
The terms and conditions page is for the main journey not the estimator, so remove "ESTIMATOR" and "-estimation" from the route:

`CREDITS_TERM_AND_CONDITIONS: '/credits/credits-terms-and-conditions`

Same for the next route